src/styles/index.css

```diff
@@ -0,0 +1,96 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
I don't understand why you connected the fonts twice. Here in css file and in html